sleap/gui/color.py

```diff
@@ -106,6 +115,14 @@ def tracks(self) -> Iterable[Track]:
         if self.labels:
             return self.labels.tracks
         return []
+    
+    @property
+    def instanceGroups(self) -> Iterable[InstanceGroup]:
```

Comment:
`snake_case` in python please (except for class names)! Although some of our code does use `camelCase`, that should only be for the code that subclasses Qt classes whose methods are written in `camelCase` (ported over from C).
```suggestion
    def instance_groups(self) -> Iterable[InstanceGroup]:
```